.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+jobs:
+  build:
+    docker:
+        - image: kaikuehne/pyenv:latest
+
+    working_directory: /tmp/app
+
+    steps:
+        - checkout
+        - setup_remote_docker
+
+        - restore_cache:
+            keys:
+                - v1-project-{{ arch }}-{{ checksum "tox.ini" }}
+                - v1-project-
+        - run:
+            command: |
+                pyenv --version
+                pyenv install --list
+                pyenv install --keep --skip-existing 2.7.14
+                pyenv install --keep --skip-existing 3.4.7
+                pyenv install --keep --skip-existing 3.5.4
+                pyenv install --keep --skip-existing 3.6.4
+                pyenv rehash
+                pyenv versions
+                pyenv local  2.7.14 3.6.4 3.5.4 3.4.7
```

Comment:
Shouldn't it support 3.3 (but not 3.6)? https://github.com/startupgrind/python-percy-client/blob/master/tox.ini#L2

Reply:
Huh.  I don't know why I did this (I think I was stumbling around the versions available on the docker image) and was more concerned about getting it to run at all.  Well...next time I poke at this.